main.lua

```diff
@@ -259,6 +260,13 @@ function karma_OnEvent(self, event, ...)
             loadMemberKarma();
             updateRaidList();
         end
+		if event == "CHAT_MSG_RAID_WARNING" then
+			local msg, author, language, channel = ...
+			if string.find(msg, "Please roll on") then

```

Comment:
Consider: Adding "KL: " to the start raid message so that people just typing "please roll on" doesn't trigger it.

Reply:
I came up with a possibly cleaner solution where it just cuts out the strings it expects and checks for an item ID in the message.